src/_common/ErrorMsg.js

```diff
@@ -1,15 +1,15 @@
 import React, { PropTypes, Component } from 'react';
 import M from '../_common/M';
+import errorToString from '../_utils/errorToString';
 
 export default class ErrorMsg extends Component {
 
 	static propTypes = {
 		shown: PropTypes.bool.isRequired,
 		text: PropTypes.string.isRequired,
 	};
-
 	render() {
 		const { shown, text } = this.props;
-		return shown ? <p className="errorfield"><M m={text} /></p> : <span />;
+		return shown ? <p className="errorfield"><M m={errorToString(text)} /></p> : null;
```

Comment:
We don't need to translate errors. 
They are already translated on the server ;)